metron-streaming/Metron-Common/src/main/java/org/apache/metron/bolt/ConfiguredBolt.java

```diff
@@ -77,15 +82,37 @@ public void childEvent(CuratorFramework client, PathChildrenCacheEvent event) th
     } catch (Exception e) {
       throw new RuntimeException(e);
     }
+
+    long timeElapsed = 0;
```

Comment:
We should load the configs directly from zookeeper synchronously rather than do a while, sleep until they get loaded asynchronously.